src/utils/apiUtils/chartData.js

```diff
@@ -204,13 +74,15 @@ function getDatesAndTimesForOneDay(
   return datesTimesAndPrices;
 }
 
+// TODO: test
```

Comment:
created https://github.com/gregdardis/stock-market-search/issues/134 to track this